src/main/java/core/basesyntax/service/impl/FruitTransactionParserImpl.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitTransactionParser;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FruitTransactionParserImpl implements FruitTransactionParser {
+    private static final String SEPARATOR = ",";
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_INDEX = 1;
+    private static final int QUANTITY_INDEX = 2;
+    
+    @Override
+    public List<FruitTransaction> getFruitTransactionsList(List<String> dataAll) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+
+        for (int i = 0; i < dataAll.size(); i++) {
+            String[] data = dataAll.get(i).split(SEPARATOR);
+            if (data[OPERATION_INDEX].equals("type")) {
+                continue;
+            }
```

Comment:
make offstep constant = 1;
```suggestion
        for (int i = OFFSTEP; i < dataAll.size(); i++) {
            String[] data = dataAll.get(i).split(SEPARATOR);
```